app/models/organization.rb

```diff
@@ -138,4 +140,8 @@ def bust_cache
   def unique_slug_including_users_and_podcasts
     errors.add(:slug, "is taken.") if User.find_by(username: slug) || Podcast.find_by(slug: slug) || Page.find_by(slug: slug) || slug.include?("sitemap-")
   end
+
+  def sync_related_elasticsearch_docs
+    DataSync::Elasticsearch::Organization.new(self, saved_changes).call
```

Comment:
now that i think about it, for future refactorings, maybe we only need `self` as `saved_changes` is a property of self anyway. Or I think :D

Reply:
Oh, well there is a refactor that needs to be done after this. I will make a note to try that!